pkg/acs/calacs/acsrej/acsrej_loop.c

```diff
@@ -137,14 +138,16 @@ Code Outline:
   12-Jan-2016   P.L. Lim    Calculations now entirely in electrons.
   14-Jan-2016   P.L. Lim    Replace threshold formula with ERR. Cleaned up
                             threshold calculation function.
+  26-Feb-2019   M.D. DeLaPena Check the IMAGETYP - can ignore cosmic ray flag (4)
+                            for BIAS and DARK images.
```

Comment:
I would switch this to "can ignore BPIXTAB flag (4) for bias and dark". This could be confusing to future people because since the cosmic ray flag is 8192

Reply:
Updated comments.